AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6511 @@
+{
```

Comment:
<blockquote>Does updating our beliefs translate into better predictions?</blockquote><p> </p><p>I dont see where this question is answered. II think you can just remove this line with no loss to the lesson</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>It is answered in the sense that posterior predictions are better than prior predictions. I added that point explicitly in the text below and overlayed the prior preds in the graph too, to make that obvious. Tell me if you still think this should be removed</p>

Reply:
<p>Ah ok, I think I would stay away from saying predictions are better because ML folks might then ask about some accuracy metric. I suggest wording this more like, we can see how data has updated our beliefs allowing us to better estimate the future</p><p> </p>